src/consensus/aft/impl/message.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include <cstdint>
+#include <limits>
+
+namespace aft
+{
+  class IMessage
```

Comment:
Minor: can we use `AbstractMessage` to be consistent with the naming of all of our interfaces so far?